jsonb/pom.xml

```diff
@@ -19,6 +19,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.avaje</groupId>
+      <artifactId>avaje-json-node</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
this should be optional no?

Reply:
or actually should it be removed?

Reply:
Ah, removed ... nice catch !!

Reply:
So I initially did it such that avaje-json-node was a dependency of jsonb ... but then I thought that wasn't so good and that it should be optional.

I think its better that avaje-json-node is optional ... but let me know if you think otherwise.